consensus/src/network_tests.rs

```diff
@@ -351,10 +351,21 @@ impl NetworkPlayground {
             .is_message_dropped(src_twin_id, dst_twin_id)
     }
 
-    pub fn drop_message_for(&mut self, src: &TwinId, dst: TwinId) -> bool {
+    pub fn drop_message_for(&mut self, src: &TwinId, dst: &TwinId) -> bool {
         self.drop_config.write().unwrap().drop_message_for(src, dst)
     }
 
+    pub fn split_network(
```

Comment:
I noticed that you return bool but it's not used?  Did you intend to assert on this in the tests?

Reply:
That was the idea :-) Added asserts.